### ISSUE TITLE
Not running as SYSTEM Error

### ISSUE THREAD

**Issue by RMNfps2** [lauralex/dwm_lut#49]
![imagem_2024-01-23_011247794](https://github.com/lauralex/dwm_lut/assets/157445458/179bf316-330a-43aa-8512-c19d1d0fb685)

This message appears when I try to apply a filter. Can someone help me ?

**Comment by lauralex** [comment]
@RMNfps2 Mmmmh, for now try to download the previous version, I have to investigate

**Comment by CobKurtain** [comment]
Same issue here and running the previous version give me a "failed to load or initialise dll"

**Comment by Crevax** [comment]
I wonder what kind of settings and/or security applications could be affecting its ability to run as SYSTEM, or if anything could change to remove that requirement. The issues must stem from inside the dwm.dll if rolling back changes the message to "failed to load or initialize dll...", but I cannot figure out what the problem area is just by looking through the code - there's too much happening that I am unfamiliar with.

If we could figure out a way to reliable replicate the issue, I could maybe step through and figure out what's going on.

**Comment by PpitoB** [comment]
I had the same problem and by downloading version v3.9.6 it works correctly. 

https://github.com/lauralex/dwm_lut/releases/tag/v3.9.6

I am running Windows 11 Pro version 23H2.

**Comment by CobKurtain** [comment]
> I had the same problem and by downloading version v3.9.6 it works correctly.
> 
> https://github.com/lauralex/dwm_lut/releases/tag/v3.9.6
> 
> I am running Windows 11 Pro version 23H2.

It's working for me too
Same W11 version

**Comment by KingCrazyC** [comment]
had to clean my computer and reinstall windows 11 for my PC to work after downloading the DWM LUT! people say that it is safe? then how come i get over 1700 registry errors????? this tool and the DWM LUT tool is NOT safe! i bet you that dogegen does not use the DWM LUT himself! he probably uses Calman or some monitor hardware 3d lut from LG... i bought Calman Studio because this shit is just not worth it..

**Comment by casef** [comment]
I very much doubt that this tool is responsible for your issues. Especially given given the issues and actions you described in other threads.

Also, Calman Studio doesn't do what this tool does (apply your LUT to the entire DWM).

**Comment by KingCrazyC** [comment]
Yeah i know Calman Studio doesn't apply to the entire DWM... i do not know why windows registry gets this issues but it does.. and windows and the registry should not be affected by ANY app or tool.. windows 11 is broken enough as it is..

**Comment by KingCrazyC** [comment]
And it puts a weird strain on my 4090.. and the performance loss is not the reason..

**Comment by Buliwyf-Stonehand** [comment]
>Yeah i know Calman Studio doesn't apply to the entire DWM

So you're paying for Calman Studio instead of using DisplayCAL for free? Or does DisplayCAL also break your [$4000 PC](https://github.com/ledoge/dwm_lut/issues/48#issuecomment-1932802205)?

**Comment by KingCrazyC** [comment]
Displaycal is totally fine! the only problem is HDR Calibration...

**Comment by KingCrazyC** [comment]
Planning on buying a LG G3 or wait for the lineup that is coming later this year :)

**Comment by casef** [comment]
There are no real issues with HDR calibration in DisplayCal outside of having to deal with a bit of a learning curve. Then again I'm willing to bet that applies to Calman Studio as well (especially given the purpose of the software).

(And yes, I was also about to ask what was supposed to be the point of pointing out the purchase of a $2000 software that is not a replacement for this utility at all and can itself be substituted by free tools, but didn't want to drag the discussion further as this is probably not the time or place for such discussions. So I apologize to anyone reading this.)

**Comment by KingCrazyC** [comment]
I had no problem with using Calman Studio! simple and easy.. and when i buy a LG G3 or something similar you know that you can apply a 3D LUT with both SDR and HDR and hardware calibrate it right? it makes HDR look much better even on the desktop.. i saw that even ledoge the creator of this tool says that he does not use this tool anymore because he uses an LG OLED with a 3D LUT...

**Comment by casef** [comment]
I would certainly hope that most people interested in 3D LUTs do understand that if you buy a display with an integrated 3D LUT feature, you can actually use that feature instead of using improvised workarounds meant for devices that don't have said feature.

**Comment by QuentinGCL** [comment]
Same error on Windows 10 22H2. The v3.9.6 it works perfectly.
